guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -9,7 +9,16 @@ There are no highlights with Katello {KatelloVersion}.
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-There are no upgrade warnings with Katello {KatelloVersion}.
+// There are no upgrade warnings with Katello {KatelloVersion}.
+
+=== Port 8443 disabled by default
+
+On {SmartProxyServer}, the deprecated port 8443 has been disabled by default on new installations and upgrades.
+Clients now rely on port 443 when communicating through a {SmartProxyServer}.
+If there are older clients still using port 8443, reconfigure the clients either by using remote execution, the katello-ca-consumer RPM, or `subscription-manager --port 443` to update `/etc/rhsm/rhsm.conf`.
+
+For users that temporarily need access to port 8443 again, the installer can be executed with `{foreman-installer} --foreman-proxy-content-reverse-proxy true`.
+This should only be used as a stop gap as port 8443 will be removed permanently in the next release.
```

Comment:
```suggestion
This should only be used as a stop gap as port 8443 will be removed permanently in the next release.
```
What does "a stop gap" mean?

Reply:
Something that is temporary because the user has no other options in that moment. And that they need to address the core issue.

Reply:
I'm not blocking the PR on this, but it would be preferred to avoid jargon and reword it.

Reply:
I cleaned it up.